Core/Core/Conversations/ConversationMessage.swift

```diff
@@ -55,6 +61,29 @@ public final class ConversationMessage: NSManagedObject, WriteableModel {
         model.mediaComment = item.media_comment.flatMap {
             MediaComment.save($0, in: context)
         }
+
+        model.participantIDs = item.participating_user_ids?.map { $0.value } ?? []
+
         return model
     }
 }
+
+extension ConversationMessage {
+    public func localizedAudience( observerID: String, userMap: [String: ConversationParticipant] ) -> String {
```

Comment:
I don't think this will always be specific to parent, so the `observerID` could more generally be called `currentUserID` or `myID`.

Reply:
I like `myID`.